autowebcompat/utils.py

```diff
@@ -188,13 +188,13 @@ def make_infinite(gen_func, elems):
 
 
 def read_labels(file_name='labels.csv'):
-    try:
+    if os.stat(file_name).st_size == 0:
+        raise ValueError('labels file is empty')
```

Comment:
Raise a normal Exception, as ValueError has a specific meaning (`Raised when an operation or function receives an argument that has the right type but an inappropriate value, and the situation is not described by a more precise exception such as IndexError.`).